[BUG] Assertion Error in testArgReduction

**Describe the bug**
When running UT on MacOS, the following errors are shown.

```
____________________________ Test.testArgReduction _____________________________
self = <mars.tensor.execution.tests.test_reduction_execute.Test testMethod=testArgReduction>
    def testArgReduction(self):
        raw = np.random.random((20, 20, 20))
    
        arr = tensor(raw, chunks=3)
    
        self.assertEqual(raw.argmax(),
                         self.executor.execute_tensor(arr.argmax())[0])
        self.assertEqual(raw.argmin(),
                         self.executor.execute_tensor(arr.argmin())[0])
    
        self.assertTrue(np.array_equal(raw.argmax(axis=0),
                        self.executor.execute_tensor(arr.argmax(axis=0), concat=True)[0]))
        self.assertTrue(np.array_equal(raw.argmin(axis=0),
                        self.executor.execute_tensor(arr.argmin(axis=0), concat=True)[0]))
    
        raw = sps.random(20, 20, density=.1)
    
        arr = tensor(raw, chunks=3)
    
        self.assertEqual(raw.argmax(),
                         self.executor.execute_tensor(arr.argmax())[0])
        self.assertEqual(raw.argmin(),
>                        self.executor.execute_tensor(arr.argmin())[0])
E       AssertionError: 1 != 2
mars/tensor/execution/tests/test_reduction_execute.py:277: AssertionError
```

Maybe some corner cases should be dealt with.

**Additional context**
Linux, Python 3.7

--- a/mars/tensor/execution/tests/test_reduction_execute.py
+++ b/mars/tensor/execution/tests/test_reduction_execute.py
@@ -271,8 +271,8 @@ class Test(unittest.TestCase):
 
         random_min = np.random.randint(0, 200)
         random_max = np.random.randint(200, 400)
-        raw[np.unravel_index(random_min, raw.shape)] = -1
-        raw[np.unravel_index(random_max, raw.shape)] = 2
+        raw_format[np.unravel_index(random_min, raw_format.shape)] = -1
+        raw_format[np.unravel_index(random_max, raw_format.shape)] = 2
 
         raw = raw_format.tocoo()
         arr = tensor(raw, chunks=3)

